src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -42,4 +38,26 @@ export default class JsonRpcNetworkClient {
     }
     return await this.axiosInstance.request(requestOptions)
   }
+
+  public async getAccountLines(
+    account: string,
+  ): Promise<AccountLinesResponseJson> {
+    // TODO: consider an option for including the 'peer' param in the request, which limits the returned trust lines to only
+    // those shared between the two accounts. (This would have to be an argument to the method here and in i-c-client too.)
+    const accountLinesRequest = {
+      method: 'account_lines',
```

Comment:
ehhhhh, currently there are only two methods we're anticipating implementing here... `account_lines` and `gateway_balances` .. so I'm inclined to say we shouldn't yet bother, especially since this value is essentially hard-coded in one place (the corresponding network client method)